docs/index.md

```diff
@@ -10,22 +10,20 @@ hide:
   <img alt="Kubescape logo" src="https://raw.githubusercontent.com/cncf/artwork/master/projects/kubescape/stacked/color/kubescape-stacked-color.svg" width="300">
 </picture>
 
-_Comprehensive Kubernetes Security from Development to Runtime_
+_Open-source Kubernetes Security: Practical, End-to-End Coverage_
```

Comment:
I like the change in the first half. However, I find "Practical, End-to_end Coverage" is not as self explanatory as ""From Development to Runtime"